test/fixtures/callbacks_no_docs.ex

```diff
@@ -0,0 +1,7 @@
+defmodule CallbacksNoDocs do
+  use Behaviour
```

Comment:
Behaviour is deprecated. Can you please use `@callback` instead of `defcallback`?